recipes/lemon/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "lemon" %}
+{% set version = "1.3.1" %}
+{% set sha256 = "71b7c725f4c0b4a8ccb92eb87b208701586cf7a96156ebd821ca3ed855bad3c8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: http://lemon.cs.elte.hu/pub/sources/lemon-{{ version  }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    - glpk_win.diff    # [win]
```

Comment:
My understanding is that on Windows the build system is trying to install the GLPK dlls as part of the lemon installation. I guess it makes kinda sense in light of their windows install instructions here:

http://lemon.cs.elte.hu/trac/lemon/wiki/InstallCmake

in which they advise to install an exernal binary package for GLPK, but clearly does not make much sense for conda.